yarn.lock

```diff
@@ -1767,6 +1767,11 @@
   resolved "https://registry.yarnpkg.com/@expo/vector-icons/-/vector-icons-14.1.0.tgz#d3dddad8b6ea60502e0fe5485b86751827606ce4"
   integrity sha512-7T09UE9h8QDTsUeMGymB4i+iqvtEeaO5VvUjryFB4tugDTG/bkzViWA74hm5pfjjDEhYMXWaX112mcvhccmIwQ==
 
+"@expo/vector-icons@^15.0.0":
```

Comment:
should we bump the version also https://github.com/expo/expo/blob/be968eaa4752ba4c833002774e55851764ac70ec/packages/expo/package.json#L85?

Reply:
thanks for catching that! 🙈